src/Core/Service/Subscriber/LicenseSyncSubscriber.php

```diff
@@ -102,33 +99,27 @@ public function serviceInstalled(AppInstalledEvent|AppUpdatedEvent $event): void
             return;
         }
 
-        try {
-            $serviceEntry = $this->serviceRegistryClient->get($app->getName());
-            $this->syncLicenseByService($serviceEntry, $app, $context);
-        } catch (\Throwable $e) {
-            $this->logger->warning('Could not sync license', ['exception' => $e->getMessage()]);
-        }
+        $this->syncLicenseByService(
+            $app,
+            $context,
+            $this->config->getString(self::CONFIG_STORE_LICENSE_KEY),
+            $this->config->getString(self::CONFIG_STORE_LICENSE_HOST)
+        );
     }
 
-    private function syncLicenseByService(ServiceEntry $serviceEntry, AppEntity $app, Context $context, ?string $licenseKey = null, ?string $licenseHost = null): void
+    private function syncLicenseByService(AppEntity $app, Context $context, string $licenseKey, string $licenseHost): void
     {
-        if ($serviceEntry->licenseSyncEndPoint === null) {
```

Comment:
since the $this->config->getString(...) can return `''` if there's no set value, should we handle that case?

Reply:
I left the logic unchanged, because previously it would also send an empty string:

```php
if ($licenseHost === null) {
    $licenseHost = $this->config->getString(self::CONFIG_STORE_LICENSE_HOST);
}
```

I would be up to do
```php
$client->syncLicense($licenseKey ?: null, $licenseHost ?: null);
```
so we explicitly send `null` over the wire if it's empty.